websites/T/Tinkercad/handleThing.ts

```diff
@@ -0,0 +1,88 @@
+// import { Assets } from "./presence";
+
+export function handleThing(
+	presenceData: PresenceData,
+	link: string,
+	isEdit: boolean
+): void {
+	if (!link) {
+		presenceData.state = "Browing the Community Gallery";
+		presenceData.buttons = [{ label: "View", url: document.location.href }];
```

Comment:
Make these buttons more specific. What do these buttons lead the user to view?

Reply:
the gallery?

Reply:
actually, I'm gonna remove it

Reply:
This comment also applies for the other buttons in this presence